libfwupdplugin/fu-common.c

```diff
@@ -1089,6 +1089,20 @@ fu_common_get_path (FuPathKind path_kind)
 		return g_strdup (EFI_APP_LOCATION);
 #else
 		return NULL;
+#endif
+	/* /usr/libexec/fwupd/efi/dbx */
```

Comment:
`libexecdir` is supposed to be executables and this stuff shouldn't be executable.  I think this should be going into `/usr/share/fwupd/` instead

Reply:
Fixed.